nixos/modules/services/databases/postgresql.md

```diff
@@ -354,6 +354,27 @@ self: super: {
 }
 ```
 
+You can define a custom postgresql extension with `postgresqlBuildExtension` for C and sql extensions, or the `buildPgrxExtension` for pgrx (rust) extensions. It is then possible to use postgresql's `callPackage` method to build the extension:
+
+```nix
+{
+  services.postgresql.extensions =
+    ps: with ps; [
+      pg_repack
+      postgis
+      (ps.callPackage ./my-extension.nix { })
+    ];
+}
+  # or
+  {
+    postgresql_custom = self.postgresql_17.withPackages (ps: [
+      ps.pg_repack
+      ps.postgis
+      (ps.callPackage ./my-extension.nix { })
+    ]);
+  }
+```
```

Comment:
I'm surprised why this doesn't fail nixfmt, which also runs on code blocks in the docs. It certainly seems to do some odd indentation (I assume that was no intended).

Maybe just split these into two code blocks?